src/entity/index.ts

```diff
@@ -36,3 +36,4 @@ export * from './SquadPublicRequest';
 export * from './UserCompany';
 export * from './Organization';
 export * from './campaign';
+export * from './Autocomplete';
```

Comment:
do we need to?

Reply:
For some reason I was getting some errors (can't exactly which script) when this was left out.